tests/dummy/app/controllers/application.ts

```diff
@@ -0,0 +1,32 @@
+import Controller from '@ember/controller';
+
+export default class Application extends Controller {
+  items = [
```

Comment:
We're not really using this, but does this work ?
```
items: {name: string}[] =
```

Reply:
this will disappear before the pr is closed. it's only dummy data to display

Reply:
Yeah I saw it was in the dummy, was just curious actually ^^